### PR TITLE
Added Cycle Name after Datapack Name in tooltips.

### LINKED ISSUE #229: Feature request - show cycle of cards in data packs on hover tooltips and card pages
Hey! I'm still pretty new to Netrunner, so I'm often not sure which cycle a card belongs to. For instance, take this hover tooltip for Reaver:

I'm not sure which cycle Reaver is in (off the top of my head, anyway), and to check I have to pull up the [search page](https://netrunnerdb.com/en/search), since it's got its nice list of sets: 

It'd be a bit easier for folks newer to Netrunner if they could see the cycle a data pack is in, like so:

> ●●●● Apex - Martial Law (Flashpoint)

Is this something you'd be open to incorporating / receiving a PR on?

### PR DESCRIPTION
**Pull request by x3N1GM4x**
Resolves #229:

image
image
image
image

### CODE REVIEW

File: web/js/nrdb.tip.js [hunk 1]
```diff
@@ -22,8 +22,16 @@
                         /.*\/card\/(\d\d\d\d\d).*/,
                         "$1"));
         var card = NRDB.data.cards.findById(code);
+
```
Comment (NoahTheDuke) [inline web/js/nrdb.tip.js]: Whitespace: 🙅‍♂️

File: web/js/nrdb.tip.js [hunk 2]
```diff
@@ -22,8 +22,16 @@
                         /.*\/card\/(\d\d\d\d\d).*/,
                         "$1"));
         var card = NRDB.data.cards.findById(code);
+        
         if(!card)
             return;
+        var cycle = NRDB.data.cycles.findById(card.pack.cycle_code);
+        var cycle_name = '';
+        if(cycle.size === 1) {
```
Comment (NoahTheDuke) [inline web/js/nrdb.tip.js]: Try `if(cycle.size !== 1) {` and then the rest. No need to reassign when you already assigned it blank above.

File: web/js/nrdb.tip.js [hunk 3]
```diff
@@ -22,8 +22,16 @@
                         /.*\/card\/(\d\d\d\d\d).*/,
                         "$1"));
         var card = NRDB.data.cards.findById(code);
+        
         if(!card)
             return;
+        var cycle = NRDB.data.cycles.findById(card.pack.cycle_code);
```
Comment (binarymelon) [inline web/js/nrdb.tip.js]: Since it appears the cycle is already defined on the card do we need to do this lookup?

**Comment by x3N1GM4x** [comment]
Ok, I think this is significantly cleaner/better, thanks for the tips!

File: web/js/nrdb.tip.js (hunk 1 above)
Comment (x3N1GM4x) [reply web/js/nrdb.tip.js]: Yes, good point! I got sloppy. 😄

**Pull request merged**